admin/config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  AZURE_STORAGE_CONNECTION_STRING: process.env.AZURE_STORAGE_CONNECTION_STRING,
+  AZURE_STORAGE_LOGGING_ENABLED: process.env.AZURE_STORAGE_LOGGING_ENABLED,
+  GOOGLE_TRACKING_ID: process.env.GOOGLE_TRACKING_ID,
+  MONGO_CONNECTION_STRING: process.env.MONGO_CONNECTION_STRING,
+  MTC_AUTH_PRIVATE_KEY: process.env.MTC_AUTH_PRIVATE_KEY,
+  MTC_SERVICE: process.env.MTC_SERVICE,
+  NCA_TOOLS_AUTH_URL: process.env.NCA_TOOLS_AUTH_URL,
+  NEW_RELIC_LICENSE_KEY: process.env.NEW_RELIC_LICENSE_KEY,
+  PORT: process.env.PORT,
+  SESSION_SECRET: process.env.SESSION_SECRET,
+  STD_LOG_FILE: process.env.STD_LOG_FILE,
+  TSO_AUTH_PUBLIC_KEY: process.env.TSO_AUTH_PUBLIC_KEY
+}
```

Comment:
I think the onus is on the config object to check that all values are populated.  If not then if the NODE_ENV is production then terminate node, if on development it could issue a warning.  We may want to differentiate between values we need on dev and those we don't.  NCA_TOOLS_AUTH_URL will never be set on dev for instance